dist/jquery.validate.unobtrusive.js

```diff
@@ -355,6 +355,14 @@
         return match;
     });
 
+    $jQval.addMethod("fileextensions", function (value, element, extensions) {
```

Comment:
Is this the same thing as the `extensions()` method in `additional-methods.js`? If yes, why aren't we using that?